src/ProjectManagement/Utility/BuildIntegratedProjectUtility.cs

```diff
@@ -79,5 +77,77 @@ public static ExternalProjectReference ConvertProjectReference(BuildIntegratedPr
         {
             return new ExternalProjectReference(reference.Name, reference.PackageSpecPath, reference.ExternalProjectReferences);
         }
+
+        public static IReadOnlyList<PackageIdentity> GetOrderedProjectDependencies(BuildIntegratedNuGetProject buildIntegratedProject)
```

Comment:
Are we absolutely positive that the scripts have to be run in dependency order? How do they behave at solution open? Isn't dependency order lost at that point? If they aren't run in dep-order then, I don't think we need to run them in dep-order here...